apps/rush/src/RushVersionSelector.ts

```diff
@@ -25,7 +26,7 @@ export class RushVersionSelector {
     configuration: MinimalRushConfiguration | undefined): Promise<void> {
 
     const isLegacyRushVersion: boolean = semver.lt(version, '4.0.0');
-    const expectedRushPath: string = path.join(this._rushDirectory, `rush-${version}`);
+    const expectedRushPath: string = path.join(this._rushDirectory, `node-${process.version}`, `rush-${version}`);
```

Comment:
>{process.version} [](start = 75, length = 17)

Can `process.version` ever have weird spaces in the name, e.g. if someone's using a weird build of NodeJS